ui/partials/perf/graphsctrl.html

```diff
@@ -101,7 +101,7 @@
           <i class="text-warning fa fa-exclamation-circle"></i>
           <a href="perf.html#/alerts?id={{tooltipContent.alertSummary.id}}">
             Alert #{{tooltipContent.alertSummary.id}}</a>
-        <span class="text-muted">- {{alertIsOfState(tooltipContent.alert, phAlertStatusMap.ACKNOWLEDGED) ? geAlertSummarytStatusText(tooltipContent.alertSummary) : getAlertStatusText(tooltipContent.alert)}}
+        <span class="text-muted">- {{tootipContent.alert && (alertIsOfState(tooltipContent.alert, phAlertStatusMap.ACKNOWLEDGED) ? geAlertSummarytStatusText(tooltipContent.alertSummary) : getAlertStatusText(tooltipContent.alert))}}
```

Comment:
There's a typo in here: should be `getAlertSummarytStatusText`

Reply:
If you're still seeing errors with the `tooltipContent.alert` condition you could remove that and add `ng-if="tooltipContent.alert"` to the span instead. But it seemed to be worked fine against stage.

Reply:
> Looks good to me. Is this still a WIP?

No, this is no longer WIP. Thanks for pointing this out.

Reply:
> There's a typo in here: should be getAlertSummaryStatusText

You're right. I actually found another one: `tooltipContent.alert`.

About the errors: there aren't any errors now. I forgot to tell you the *.html changes fixed the errors.

Reply:
Ok. It's easier to see what code needs to be reviewed if you keep your changes as separate commits instead of rebasing/squashing them.